bin/pycbc_optimize_snr

```diff
@@ -493,7 +496,7 @@ if args.enable_gracedb_upload:
     comment = ('Automatic PyCBC followup of trigger '
                '<a href="/events/{0}/view">{0}</a> to find the template '
                'parameters that maximize the SNR. The FAR of this trigger is '
-               'copied from {0} and does not reflect this triggers\' template '
+               'copied from {0} and does not reflect this trigger\'s template '
```

Comment:
Should the comment say the same for p_astro?

Reply:
Dealing with pastro for the SNR max upload is beyond the scope of this PR